### ISSUE TITLE
Change output layer to use softmax

### ISSUE THREAD

**Issue by Octogonapus** [Octogonapus/MoralAI#4]
Instead of having one output neuron, there should be two and the `softmax` activation function should be used. The first neuron will represent the first option, etc. This might also let us use the `categorical_crossentropy` loss function.
